docs/diagrams/CommandsDiagram.puml

```diff
@@ -0,0 +1,59 @@
+@startuml
+!include Styles.puml
+
+Package "Commands                    "  <<Rectangle>>{
+     Class "{abstract}\nCommand" as Command COLOR_COMMANDS {
+     {method} + setData
+     {method} + setParams
+     {method} + execute: String
+     }
+     Class ViewCommand COLOR_COMMANDS
+     Class EditInstrumentCommand COLOR_COMMANDS
+     Class DoneCommand COLOR_COMMANDS
+     Class DeleteCommand COLOR_COMMANDS
+
+     Class InvalidCommand COLOR_COMMANDS
+     Class ABCCommand COLOR_COMMANDS
+     Class AddInstrumentCommand COLOR_COMMANDS
+     Class "{abstract}\nIndexedCommand" as IndexedCommand COLOR_COMMANDS
+
+     Class AddXYZCommand COLOR_COMMANDS
+}
+
+
+Class InstrumentManager COLOR_MODEL {
+    {method} + getSize: int
+    {method} + getInstruments: ArrayList<Instrument>
+    {method} + getInstrument: Instrument
+    {method} + addInstrument: Instrument
+    {method} + findInstruments: ArrayList<Instrument>
+    {method} + deleteInstrument
+    {method} + editInstrument
```

Comment:
All the methods need a bracket with the type of parameters they take in
```suggestion
    {method} + getSize(): int
    {method} + getInstruments(): ArrayList<Instrument>
    {method} + getInstrument(int): Instrument
    {method} + addInstrument(Instrument): Instrument
    {method} + findInstruments(String): ArrayList<Instrument>
    {method} + deleteInstrument(int)
    {method} + editInstrument(int, Hashmap<String, String>)
```

Reply:
I do not think this is a standard uml convention. Can you show me a class diagram on the website which uses such a convention of having the arguments' types in parantheses?

Reply:
<img width="546" alt="ss1" src="https://user-images.githubusercontent.com/60163283/140603574-d5ec101f-e6e9-47fe-ae00-8e1b5b576eb8.PNG">